### PR TITLE
Update validators.da.xlf

### LINKED ISSUE #38736: Missing translations for Danish (da)
Hello,

There are some missing translation for Danish (da). These should be added in branch 4.4. 
See https://github.com/symfony/symfony/issues/38710 for details and [this page](https://symfony-translations.nyholm.tech/#pr) for an example.

These are the files that should be updated: 
- [src/Symfony/Component/Form/Resources/translations/validators.da.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Form/Resources/translations/validators.da.xlf)
- [src/Symfony/Component/Security/Core/Resources/translations/security.da.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Resources/translations/security.da.xlf)
- [src/Symfony/Component/Validator/Resources/translations/validators.da.xlf](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Validator/Resources/translations/validators.da.xlf)

NOTE: If you want to work on this issue, add a comment to assign it to yourself and let others know that this is already taken.

### PR DESCRIPTION
**Pull request by AnneKir**
Added missing cases in danish.

| Q             | A
| ------------- | ---
| Branch?       | 4.4 
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no
| Tickets       | Fix #38736 
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

**Comment by AnneKir** [comment]
Is it possible to label this with  "hacktoberfest-accepted"?

File: src/Symfony/Component/Form/Resources/translations/validators.da.xlf [hunk 1]
```diff
@@ -12,7 +12,127 @@
...
+                <source>Please enter an integer.</source>
+                <target>Indsæt veligst et heltal.</target>
+            </trans-unit>
+            <trans-unit id="112">
+                <source>Please select a valid language.</source>
+                <target>Vælg venligst et gyldigt sprog.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Vælg venligst et gyldigt landestandard.</target>
```
Comment (Nyholm) [inline src/Symfony/Component/Form/Resources/translations/validators.da.xlf]: Note that this is not a "country standard" it is a "language standard". 

The country code for Denmark is "DK" but the language code for Danish is "DA". Same with Sweden, SE and SV

**Review by Nyholm — COMMENTED** [review]
Thank you for this change. 

I just have one small thing that I would like you to confirm. 

-------

We have the "hacktoberfest" topic, so your contributions will count. 

Screenshot 2020-11-01 at 11 26 22

File: src/Symfony/Component/Form/Resources/translations/validators.da.xlf (hunk 1 above)
Comment (AnneKir) [reply src/Symfony/Component/Form/Resources/translations/validators.da.xlf]: I see. I have now changed it to "language code".

**Review by Nyholm — APPROVED** [review]
Awesome. Im happy with this 👍 

Thank you

**Comment by derrabus** [comment]
Thank you @AnneKir.

**Comment by Nyholm** [comment]
Cool. 

Thank you @AnneKir!

**Pull request merged**